BUILD

```diff
@@ -50,3 +50,16 @@ experimental_test_shell_command(
     tools=["echo", "find"],
     execution_dependencies=[":all-__init__.py-files"],
 )
+
+docker_environment(
+    name="docker_env_for_testing",
+    image="debian:stable-slim",
+    mounts=["/etc/passwd:/mount_dir/testfile:ro"],
+)
+
+experimental_test_shell_command(
+    name="test-docker-environment-bind-mounts",
+    tools=["test"],
+    command="test -f /mount_dir/testfile",
+    environment="docker_for_testing"
+)
```

Comment:
Might this be better placed in `src/python/pants/core/util_rules/BUILD` since it's testing environment functionality and that's where `environments.py` is?

Reply:
Good idea! I don't like polluting this build file either